src/main/java/com/wavefront/sdk/jersey/WavefrontJerseyFilter.java

```diff
@@ -318,4 +388,91 @@ private AtomicInteger getGaugeValue(MetricName metricName) {
       put("jersey" + ".resource.method", finalMethodName);
     }};
   }
+
+  private SpanContext parentSpanContext(ContainerRequestContext requestContext) {
+    Span activeSpan = tracer.activeSpan();
+    if (activeSpan != null) {
+      return activeSpan.context();
+    } else {
+      return tracer.extract(
+              Format.Builtin.HTTP_HEADERS,
+              new ServerHeadersExtractTextMap(requestContext.getHeaders())
+      );
+    }
+  }
+
+  private void decorateRequest(ContainerRequestContext requestContext, Span span) {
+    Tags.COMPONENT.set(span, JERSEY_SERVER_COMPONENT);
+    Tags.HTTP_METHOD.set(span, requestContext.getMethod());
+    String urlStr = null;
+    URL url;
+    try {
+      url = requestContext.getUriInfo().getRequestUri().toURL();
+      urlStr = url.toString();
+    } catch (MalformedURLException e) {
+      // ignoring returning null
+    }
+    if (urlStr != null) {
+      Tags.HTTP_URL.set(span, urlStr);
+    }
+  }
+
+  private void decorateResponse(ContainerResponseContext responseContext, Span span) {
+    Tags.HTTP_STATUS.set(span, responseContext.getStatus());
+    Tags.ERROR.set(span, isErrorStatusCode(responseContext));
```

Comment:
I think error tag should be present only when, error occurs. 
@sushantdewan123 do you see any specific use case on being able to filter error=false? It may be helpful to search for successful spans.